@css/main_style.css

```diff
@@ -926,3 +917,50 @@ dl.tech-menu > dt {
   font-weight: 500;
   line-height: 24px;	/* 120% */
 }
+
+#form-name {
+  background-color: gray;
+  color: #172b4d;		/* #172B4D */
+  font-family: Poppins, sans-serif;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;	/* 117.647% */
+}
+
+#form-name:invalid {
+  background-color: red;
+  color: white;
+}
+
+#form-name:valid {
+  background-color: white;
+}
+
+#form-email {
+  background-color: gray;
+}
+
+#form-email:invalid {
+  background-color: red;
+  color: white;
+}
```

Comment:
- [ ] I understand that you want to set the background to red when the user inputs an invalid value. However, the behaviour of your form inputs is not as intended. Before the user can enter their inputs, the form inputs are already set to "red" 🔴  which doesn't match the Figma design. And note that the input background is not "gray" 🍇 in the Figma template.
- [ ] [OPTIONAL] Remark the line at the bottom of the page in the Figma design, yours is missing. However, one of the core requirement of this project is to translate UX/UI from Figma.

Your design before user inputs 👇🏽 

![Screenshot from 2023-07-21 18-53-28](https://github.com/Microverse-ajgomes/Personal-Portfolio/assets/10439283/cc95ecb9-85d4-4f09-9a96-d2c78f9ce30e)

Figma design 👇🏽 

![Screenshot from 2023-07-21 19-19-14](https://github.com/Microverse-ajgomes/Personal-Portfolio/assets/10439283/a2773c2f-4516-477b-a6d9-1ddaa73a8544)